packages/rocketchat-emoji/client/emojiParser.js

```diff
@@ -49,6 +52,9 @@ callbacks.add('renderMessage', (message) => {
 
 		// apostrophe (') back to &#39;
 		message.html = message.html.replace(/\'/g, '&#39;');
+
+		// apostrophe ' <br> ' back to '<br>'
+		message.html = message.html.replace(/ <br> /g, '<br>');
```

Comment:
wondering if this could break other things as it will change *all* ` <br> ` to `<br>` and not *only* the previously changed ones.

Reply:
Initally all the '<br>' will change to ' <br> ' irrespective of anything in the input message. We are just reversing the process in which all the ' <br> ' will be reverted back to '<br>' in the input message.

Reply:
Initiall _all_ `<br>` were converted to ` <br> ` and in this step we are reversing the process. So the input message remains the same.

Reply:
@sampaiodiego Requesst for FInal review.

Reply:
right, that previous comment is obviously true :stuck_out_tongue: